### ISSUE TITLE
Add QC to RNA-seq workflow

### ISSUE THREAD

**Issue by jasonwalker80** [genome/analysis-workflows#186]
Run Picard CollectRnaSeqMetrics on the aligned CRAM file to generate basic RNA-seq QC metrics. This will require converting the GTF to a refFlat format and also providing an intervals format for Ribosomal and Mitochondrial annotation.

**Comment by jasonwalker80** [comment]
Example: ```gmt gtf to-ref-flat --input-gtf-file=/gscmnt/gc2560/core/GRC-human-build38_human_90_38_U2AF1_fix/rna_seq_annotation/Homo_sapiens.GRCh38.90.gtf --output-file=/gscmnt/gc2560/core/GRC-human-build38_human_90_38_U2AF1_fix/rna_seq_annotation/Homo_sapiens.GRCh38.90.refFlat.txt```

```
grep rRNA /gscmnt/gc2560/core/GRC-human-build38_human_90_38_U2AF1_fix/rna_seq_annotation/Homo_sapiens.GRCh38.90.gtf > /gscmnt/gc2560/core/GRC-human-build38_human_90_38_U2AF1_fix/rna_seq_annotation/Homo_sapiens.GRCh38.90.rRNA.gtf

gmt gtf to-intervals --gtf-file=/gscmnt/gc2560/core/GRC-human-build38_human_90_38_U2AF1_fix/rna_seq_annotation/Homo_sapiens.GRCh38.90.rRNA.gtf --interval-file=/gscmnt/gc2560/core/GRC-human-build38_human_90_38_U2AF1_fix/rna_seq_annotation/Homo_sapiens.GRCh38.90.rRNA.interval_list --seqdict-file=/gscmnt/gc2560/core/model_data/2887491634/build21f22873ebe0486c8e6f69c15435aa96/seqdict/seqdict.sam

```

**Comment by jasonwalker80** [comment]
Using `mgibio/cle` docker image:
```
/usr/bin/java -Xmx16g -jar /opt/picard/picard.jar CollectRnaSeqMetrics REF_FLAT=/gscmnt/gc2560/core/GRC-human-build38_human_90_38_U2AF1_fix/rna_seq_annotation/Homo_sapiens.GRCh38.90.refFlat.txt RIBOSOMAL_INTERVALS=/gscmnt/gc2560/core/GRC-human-build38_human_90_38_U2AF1_fix/rna_seq_annotation/Homo_sapiens.GRCh38.90.rRNA.interval_list STRAND=SECOND_READ_TRANSCRIPTION_STRAND  O=test_rna_metrics.txt CHART=test_rna_metrics.pdf I=/gscmnt/gc2761/aml_ppg/model_data/4ac3f004141445adba90fd51965a2926/build82e1f36656624a1db412e7e6318b2ed4/results/merged.bam
```

**Comment by jasonwalker80** [comment]
Closed by #326.
